package.json

```diff
@@ -1,39 +1,37 @@
 {
-  "name": "glsl-pipeline",
-  "version": "1.0.6",
-  "description": "Prototype complex pipelines directly from a single shader by branching it into stages.",
+  "name": "glsl-pipeline-monorepo",
+  "private": true,
```

Comment:
What it means here to be `private` ?

Reply:
Absolutely! The root folder wouldn't publish to npm package due to its private package. Only public packages & allow public access to be publish on npmjs. Simply run the command as I suggested 😁

Reply:
fantastic